modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -144,14 +67,19 @@ For example, to run it every hour at 47 minutes past the hour, add the following
 
 NOTE: Please avoid any multiples of 10, because those are when the ClamAV servers are hit the hardest for updates.
 
-== Install the Anti-Virus App
+== Install the ownCloud Anti-Virus App
 
 The Anti-Virus app needs to be installed from the ownCloud Market (it's available in the  _ "Security"_ category).
-You can access the ownCloud Market via the App Menu (or App Switcher).
 
 image:apps/files_antivirus/antivirus-app.png[image]
 
-TIP: The Anti-Virus app can also be downloaded, installed, and enabled manually.
+--------
+
+To install the App via occ command, execute:
+
+----
+occ market:install files_antivirus
```

Comment:
Please use `{occ-command-example-prefix}` instead of `occ`

Reply:
discussed with the support team - this adds only the "sudo -u www-data" to the command which is no necessary if the user / customer followed our installation guide and has the wrapper script.

Reply:
If a "user" has more than one instance on the same system, this does not work anymore. It would be better to have a statement in your docs that if the wrapper is used, no preceedings are necessary.
All occ commands shouldbe written according the example provided to be in line with the tons of examples in docs.